### ISSUE TITLE
Weird names for missing individuals in pedigree pics

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/cg#482]
Look at this case for instance: https://scout.scilifelab.se/cust002/F0015186

Missing individuals look like this:

![image](https://user-images.githubusercontent.com/28093618/64944578-f2061d80-d86e-11e9-9c50-28596ae32537.png)

Not super urgent but it's a bug I think

**Comment by emiliaol** [comment]
I've added this issue to the backlog now, please remember to do that in the future. Thanks!

**Comment by Mropat** [comment]
@henrikstranneheim sould this be completed in current version of cg?

**Comment by ChrOertlin** [comment]
@dnil, @northwestwitch is this still an issue today?

**Comment by northwestwitch** [comment]
Yes I think so, haven't seen these weird names in ages. let's close

**Comment by dnil** [comment]
It's still there - see e.g. https://scout.scilifelab.se/cust002/F0061382.

**Comment by northwestwitch** [comment]
> It's still there - see e.g. https://scout.scilifelab.se/cust002/F0061382.

Right! Let's keep it open then, yes!

**Comment by ChrOertlin** [comment]
Ok, thanks. I will add it to refinement. 19-06-2024

**Comment by Vince-janv** [comment]
I made the mistake to spend time on this... Basically Madeleine assigns "virtual" ids to missing parents. These are for some reason always showed unless you add labels with the -L flag. Basically we can get this quite easily:
<img width="226" alt="Screenshot 2024-06-13 at 16 21 50" src="https://github.com/Clinical-Genomics/cg/assets/69356202/9d01b41c-0d52-4911-937d-e767cf8a8fe6">

**Comment by ChrOertlin** [comment]
@dnil , @northwestwitch Given the information above. Would you be happy with giving an -L flag, und just have the "ghost" box without a virtual id?

**Comment by dnil** [comment]
Heh, not having odd confusing strings there is a low prio Good Thing. The ideal would obviously be the plot from @Vince-janv without the extra label, but every little helps, and something that always appears would soon be ignored. Does it have to be called "IndividualId" by the way? Is that from StatusDB or just Madeleine?

**Comment by ChrOertlin** [comment]
I checked with vincent and it seems that when you set a label no nothing, the box appears. From the documentation the shown grey box is what would be displayed. And it is a Madeleine, and not a CG, thing.

**Comment by Vince-janv** [comment]
Yeah `IndividualId` is the name of the Madeleine label. As this will show up in all pedigrees and missing parents are a minority, maybe it is a smaller cost to have a weird string in a few cases than a confusing label in all of them?

**Comment by Vince-janv** [comment]
Lacking a better option than the two mentioned above I vote for keeping the current option. Going in and editing the file is to me out of question. @dnil Re-open if you disagree
